src/main/java/nl/tudelft/contextproject/model/VRPlayer.java

```diff
@@ -54,4 +93,62 @@ public void mapDraw(Graphics2D g, int resolution) {
 	public void setSpatial(Spatial spatial) {
 		this.spatial = spatial;
 	}
+
+	/**
+	 * Set character control.
+	 * @param characterControl
+	 * 				control to set
+	 */
+	public void setCharacterControl(CharacterControl characterControl) {
+		this.playerControl = characterControl;
+	}
+	
+	/**
+	 * Get the player hit box.
+	 * 
+	 * @return	
+	 * 				player physics object
+	 */
+	@Override
+	public CharacterControl getPhysicsObject() {
+		if (spatial == null) {
+			this.getSpatial();
+		}
+		if (playerControl != null) return playerControl;
+		//create a shape that implements PhysicsControl
+		CapsuleCollisionShape capsuleShape = new CapsuleCollisionShape(PLAYER_RADIUS, PLAYER_HEIGHT, PLAYER_AXIS);
+		playerControl = new CharacterControl(capsuleShape, PLAYER_STEP_HEIGHT);
+
+		//Add physical constants of player
+		playerControl.setJumpSpeed(JUMP_SPEED);
+		playerControl.setFallSpeed(FALL_SPEED);
+		playerControl.setGravity(PLAYER_GRAVITY);
+
+		//set physics location of player
+		playerControl.setPhysicsLocation(spatial.getLocalTranslation());
+		return playerControl;
+	}
+
+	/**
+	 * On button press, what the player should do.
+	 * @param name
+	 * @param isPressed
+	 * @param tpf
+	 */
+	@Override
+	public void onAction(String name, boolean isPressed, float tpf) {
+		if (name.equals("Left")) {

```

Comment:
Could potentially be a string switch.
(If people agree, in eclipse this can be changed automatically: Select the entire if + else ifs, press ctrl + 1, choose convert to switch)